usaspending_api/api_docs/api_documentation/advanced_award_search/spending_by_transaction.md

```diff
@@ -0,0 +1,82 @@
+## [Spending by Transaction](#spending-by-transaction)
+**Route:** `/api/v2/search/spending_by_transaction/`
+
+**Method:** `POST`
+
+This route takes keyword search fields, and returns the fields of the searched term.
+
+### Request
+field: Defines what award variables are returned.
```

Comment:
The parameter is `fields`.

fields: An array of string column names to return. See Fields list below.

Reply:
done